src/tests/api/auth/sign-in/validate.spec.ts

```
@@ -0,0 +1,285 @@
+import { validate } from "api/auth/sign-in/validate";
+import { CustomError } from "utils/error";
+import { StatusCodeEnum } from "enums/status-code";
+
+describe("auth > sign-in > service", () => {
+	const validEmail = "johndoe@foo.foo";
+	const validPassword = "JohnDoe@123";
+
+	describe("Success", () => {
+		it("should return validated params", async () => {
+			let result: any;
+
+			try {
+				result = await validate({ email: validEmail, password: validPassword });
+			} catch (err) {
+				result = err;
+			}
+
+			expect(result).toStrictEqual({
+				email: validEmail,
+				password: validPassword,
+			});
+		});
+	});
+
+	describe("Invalid Params", () => {
+		it("should throw error with invalid password have only lowercase", async () => {
```

`it("should throw an error with invalid password (whitout uppercase characters)"`